components/admin/widget/form/steps/Step2.js

```diff
@@ -146,7 +151,18 @@ class Step2 extends React.Component {
 Step2.propTypes = {
   form: PropTypes.object,
   dataset: PropTypes.string.isRequired,
-  onChange: PropTypes.func
+  onChange: PropTypes.func,
+  // REDUX
+  widgetEditor: PropTypes.object,
+  setTitle: PropTypes.func
 };
 
-export default Step2;
+const mapStateToProps = state => ({
+  widgetEditor: state.widgetEditor
+});
+
+const mapDispatchToProps = dispatch => ({
+  setTitle: title => dispatch(setTitle(title))
+});
+
+export default connect(mapStateToProps, mapDispatchToProps)(Step2);
```

Comment:
I'm afraid that we are not using anymore the `admin/widget` folder. So....this change is unnecessary